acid/features/zuul_manager/manager.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+import shlex
+
+import paramiko
+
+from .exceptions import ZuulManagerConfig, ZuulManagerConn
+
+
+class ZuulManager:
+    def __init__(self, host, username, user_key_file, host_key_file,
+                 tenant, trigger, project, policy):
+        self.host = host
+        self.username = username
+        self.host_pub_key = host_key_file
+        self.policy = policy
+
+        self.tenant = tenant
+        self.trigger = trigger
+        self.project = project
+
+        try:
+            self.user_key = paramiko.RSAKey.from_private_key_file(user_key_file)
+            self._client = self._prepare_client(self.host_pub_key)
+        except FileNotFoundError as exc:
+            raise ZuulManagerConfig() from exc
+
+    def enqueue(self, pipeline, branch):
+        command = str(f"zuul enqueue-ref --tenant {shlex.quote(self.tenant)}"
+                      f" --trigger {shlex.quote(self.trigger)}"
+                      f" --pipeline {shlex.quote(pipeline)}"
+                      f" --project {shlex.quote(self.project)}"
+                      f" --ref {shlex.quote(f'refs/head/{branch}')} &")
```

Comment:
how about: ''' blah blah blah 
                      blah blah blah
                      blah blah blah '''?

 Using ''' ''' instead of str() ?